### PR TITLE
Display board and check if moves valid 

### PR DESCRIPTION
**Pull request by fi-ya**
Tested behaviour first, so program is able to generate a board with either X or 0. Then tested for correct/valid input(0-8). Currently blocked at testing to see whos turn it is. I created turn_count method but getting back 'NoMethodError'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+- [ ] A new repository that is checked into version control and accessible to the mentoring team.
+- [ ] Any relevant packages are managed by a package manager rather than copy/pasted into the repo.
+- [ ] A single passing test (this might be a dummy test that asserts true equals true).
+- [ ] A detailed README outlining how to run the app and tests.
+- [ ] A continuous integration service that runs the test suite upon every commit. One suggestion is to use GitHub and GitHub Actions.
+- [ ] A simple message that is displayed on the console when the app is run (e.g. "Welcome to Tic-Tac-Toe").
+
+## Run game
+
+## Run tests
```
Comment (Himalee) [inline README.md:32]: Could we add the command here that runs the tests?

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        if player == "X" && valid_move?(index)
+            place_player("X", index)
+        elsif player == "0" && valid_move?(index)
+            place_player("0", index)
+        else
+            "move not vaild"
+        end
+    end
+
+    def position_taken?(index)
```
Comment (Himalee) [inline lib/board.rb:31]: We can simplify this to method to just return the result of 
```board[index] == "X" || board[index] == "0"
```
which will be true or false.

File: spec/board_spec.rb [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            output = @new_board.generate_board()
+            expect(output).to eq("   |   |   \n-----------\n   |   |   \n-----------\n   |   |   \n")
+        end   
+ 
+        it "should display correct player marker in the top left corner" do
+            verify_top_left('X', 0)
+            verify_top_left('O', 0)
+        end
+
+        def verify_top_left(player, index)
```
Comment (Himalee) [inline spec/board_spec.rb:19]: Nice refactoring this out into a test helper method! We can move the helper method to the bottom of the file so the focus of the file remains the tests.

**Review by Himalee — APPROVED** [review]

File: lib/board.rb (hunk 2 above)
Comment (fi-ya) [reply lib/board.rb:31]: Thanks! I didn't know it was possible to refactor it like that.

**Pull request merged**